packages/components/src/components/Drawer/DrawerHeading.tsx

```diff
@@ -13,7 +13,7 @@ const DrawerHeading = React.forwardRef<HTMLHeadingElement, DrawerHeadingProps>(
     return (
       <Heading
         as="h2"
-        color="colorTextHeadingStronger"
+        color="contentPrimary"
```

Comment:
I think we don't even need to pass the color, right?

Reply:
you are absolutely right! I'll remove it!